roles/openshift_manage_node/tasks/config.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Set node schedulability
+  oc_adm_manage_node:
+    node: "{{ openshift.node.nodename | lower }}"
+    schedulable: "{{ 'true' if l_openshift_manage_schedulable | bool else 'false' }}"
+  retries: 10
+  delay: 5
+  register: node_schedulable
+  until: node_schedulable is succeeded
+  when: "'nodename' in openshift.node"
+  delegate_to: "{{ openshift_master_host }}"
+
+- name: Label nodes
```

Comment:
We can combine the two label tasks instead of having the nodes skip an extra task.

```yaml
labels: "{{ l_label_to_apply | lib_.. }}"
...
vars:
  l_label_to_apply: "{{ 'oo_masters_to_config' in group_names | bool | ternary(openshift_node_labels | combine(openshift_master_node_label), openshift_node_labels) }}"
```

May need to work out the ternary statement by hand to ensure it's being cast to bool correctly; Also we should change the variable 'openshift_master_node_label' to '.._labels' because we might (or user might) add more labels in the future.

Reply:
Renamed the var to `openshift_master_node_label`

Not sure if its a good idea to use a complex `combine ..` statement. In order to comply with upstream standards the nodes would have to be labeled - and infra nodes too. This would make the statement way too complex

Reply:
I don't have a strong preference on how we do it as long as we're not adding a task that's going to be skipped by nodes by default.

I don't think the combine statement is really that complex.  I was just giving one example on how we could do it; we don't need to combine masters and nodes into the same task file if they are both getting their own tasks anyway.

Reply:
Reworked the task to combine labels in one task, PTAL

Reply:
That looks good.  Thanks for taking the time to implement this.